index.html

```diff
@@ -7614,7 +7614,8 @@ <h2 id="AudioWorklet-Examples">
             <p>
               This method is DEPRECATED. It is equivalent to setting
               <code>positionX</code>, <code>positionY</code>, and
-              <code>positionZ</code> AudioParams directly.
+              <code>positionZ</code> <a>AudioParam</a>s <code>value</code>
+              attribute directly.
```

Comment:
Not your fault, but that's pretty vague on what "setting" means.  If we clarify this to say the same as `positionX.setValueAtTime(x, context.currentTime)` and so on, would we need the rest of the text below?  I think if we say this, it makes it pretty clear what happens if there's an `setValueCurveAtTime` there.

Reply:
I've decided to copy some prose I found for setOrientation, and make all that extra clear.